src/Extension/Traits/FluentAdminTrait.php

```diff
@@ -663,4 +670,56 @@ protected function updateDeleteAction(FieldList $actions, DataObject $record): v
                 )
             );
     }
+
+    /**
+     * Information panel shows published state of a base record by default
+     * this overrides the display with the published state of the localised record
+     *
+     * @param FieldList $actions
+     * @param DataObject|FluentVersionedExtension $record
+     */
+    protected function updateInformationPanel(FieldList $actions, DataObject $record): void
+    {
+        if (!$record->hasExtension(Versioned::class)) {
+            // This is only relevant for versioned records (base)
+            return;
+        }
+
+        if (!$record->hasExtension(FluentVersionedExtension::class)) {
+            // This is only relevant for versioned records (fluent)
+            return;
+        }
+
+        /** @var Tab $moreOptions */
+        $moreOptions = $actions->fieldByName('ActionMenus.MoreOptions');
```

Comment:
My understanding is that MoreOptions is for sitetree only, right?

Do you want to soft-support non-sitetree models too?

Reply:
Do non-sitetree objects have Information panel?

Reply:
yeah, they do. It's a relatively recent addition though.

https://github.com/silverstripe/silverstripe-versioned/blob/1.7/src/VersionedGridFieldItemRequest.php#L308